x/simulation/simulate.go

```diff
@@ -31,8 +31,12 @@ func initChain(
 	config simulation.Config,
 	cdc codec.JSONCodec,
 ) (mockValidators, time.Time, []simulation.Account, string) {
+	blockMaxGas := int64(-1)
+	if config.BlockMaxGas != nil {
+		blockMaxGas = *config.BlockMaxGas
+	}
```

Comment:
ditto

Reply:
Why can't the default be `-1` w/o using a pointer?

Reply:
[updated](https://github.com/cosmos/cosmos-sdk/pull/16656/commits/86b7150d114937e1dcb28099a10cf480d802f693), but we can't set maxGas as 0 unless pointer